OPAL/br/src/main/scala/org/opalj/br/Type.scala

```diff
@@ -1519,6 +1560,39 @@ final class ArrayType private ( // DO NOT MAKE THIS A CASE CLASS!
  */
 object ArrayType {
 
+    /**
+     *  Flushes the global cache for ArrayType instances. This does not include the predefined types,
+     *  which are kept in memory for performance reasons.
+     */
+    def flushTypeCache(): Unit = {
```

Comment:
Should we have a method (probably on ReferenceType) that invokes both this and the ObjectType cache flush to do both at once?

Reply:
Yes that's a good idea. I was thinking about where to this method, but ReferenceType makes perfect sense to me. Will do right away 👍